modules/apps/headless/headless-admin-content/headless-admin-content-impl/src/main/java/com/liferay/headless/admin/content/internal/resource/v1_0/StructuredContentResourceImpl.java

```diff
@@ -1,11 +1,11 @@
 /**
  * Copyright (c) 2000-present Liferay, Inc. All rights reserved.
- *
+ * <p>
```

Comment:
What happened here? Source formatter?

Reply:
My bad. I use a shortcut in IntelliJ to refactor the code and adds the `<p>` in the Copyright. I have to change the configuration in my IJ to avoid that 😓

Reply:
ok